theme/templates/resource-landing-page/resource-header.html

```diff
@@ -105,31 +108,28 @@
                 </tr>
                 <tr>
                     <th>Created:</th>
-                    <td><span class="local-datetime" data-dt="{{ cm.created|date:'c' }}"></span></td>
+                    <td>{{ cm.created|date:"M d, Y" }} at {{ cm.created|time }} UTC</td>
```

Comment:
this looks pretty good!
I am noting that the original request was to append `(UTC)` and we are appending `UTC` without parenthesis.
![image](https://github.com/user-attachments/assets/3948e4a1-841d-4c03-a33a-35e253e4acea)